.vscode/settings.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I wonder if we can be more `DRY`?  There is a lot of duplication across the env vars defined here and the env vars in the `docker-compose` files.  Does it make sense to have a common env var file that is read in (by both `docker-compose` and `vscode`)?  Or does it make sense to isolate the environments and have the env vars explicitly defined (like it is set up now)?